### DISCUSSION TITLE
HTML and Custom widget links in Preview Mode lead to pages without the site name

### DISCUSSION THREAD

**Discussion by mariosm1** [percussion/percussioncms#1058] [Troubleshooting]
**Describe the bug**
When previewing a page in the cms - if you follow a link on that page, it will take you to the correct page but without the /Sites/<sitename> prefix.

e.g.

- Previewing http://cm1.percussion.com:9992/Sites/testsite/index.html
- Click a link that was created in a custom widget or HTML widget (<a href="/otherpage.html">Other Page</a>)
- You will be navigated to http://cm1.percussion.com:9992/otherpage.html

**Expected behavior**
HTML and custom widget links should correctly add the /Sites/<sitename> prefix when navigating pages in the CMS Preview Mode

**Screenshots**
![im1](https://user-images.githubusercontent.com/32443983/235757139-9c65ef0a-f00f-4408-8c06-dcc4e81bdffc.png)
![im2](https://user-images.githubusercontent.com/32443983/235757157-06d0216b-2a51-4d1e-bd61-2f66a720f83e.png)

**Comment by natechadwick** [comment]
This isn't really a bug.  In order for Preview to work correctly the link should be updated to match the site path:

``
<a href="/Sites/mysite/index.html">Home</a>
``

The system will convert the link into a [managed link](https://percussioncmshelp.intsof.com/percussion-cm1/manage-content/managed-links/) when it is saved.  This will allow Preview to work, and the system will change the link automatically to /index.html when the page is published.  The main advantage to this other than Preview working, is that if the page is moved or renamed, the system will track the change and updated any managed links that pointed to the old page location automatically.
